pkg/daemon/server.go

```diff
@@ -141,16 +145,18 @@ func newHub() *Hub {
 
 // In the end we need to represent a message we want to send to a connected client.
 // The only sensible way we can achive that without loosing too much information is to encode the entire
-// object as json.
-func (m *message) Bytes() []byte {
-	body, _ := json.Marshal(m)
+// array of messages as json array.
+func (e *envelop) Bytes() []byte {
+	body, _ := json.Marshal(e.messages)
 	return body
 }
 
 // Send a message to single user on all his connections
 // This API is probably a little volatile so use with caution and don't reach deep into it.
 func (h *Hub) broadCastTo(u *UserID, topic Topic, data interface{}) {
-	h.broadcast <- &envelop{u, message{topic, data}}
+	var messages []message
+	messages = append(messages, message{topic, data})
```

Comment:
No, not yet, but it will teach the front end to loop the resulting structure and dispatch events from it. What still needs to happen is to implement a debouncer that handles packing messages. But that IMO can happen in another MR. As these changes already enable the implementation and force the needed changes on the frontend, that then is able to digest all coming changes. Makes sense?